code/modules/clothing/spacesuits/rig/modules/ai.dm

```diff
@@ -245,6 +245,58 @@
 		"The suit power is almost dead, can't help you much",
 		)
 
+/datum/rig_aivoice/MK_2
+	name = "Mk II"
+	welcome_message = "Hello. It's nice to meet you"
+	damage_message = "Ouch. I am not used to feeling pain"
+	destroyed_message = "Hello... anyone......"
+
+	health_warnings = list(
+		"I've found some scrathes",
+		"Caution, you're not looking good",
+		"It would be nice for you, to leave the area you are in",
```

Comment:
"It would be nice for you, to leave the area you are in"

Запятая не должна быть в середине предложения.

"It would be nice for you to leave the area you are in"